mta/mta_services_test.go

```diff
@@ -1426,6 +1426,28 @@ var _ = Describe("MtaServices", func() {
 			Ω(messages).Should(BeEmpty())
 		})
 
+		It("Get build parameters using extension file", func() {
+			myBuilder := ProjectBuilder{
+				Builder: "mybuilder",
+			}
+			otherBuilder := ProjectBuilder{
+				Builder: "otherbuilder",
+			}
+			oBuildParameters := ProjectBuild{
+				BeforeAll: []ProjectBuilder{myBuilder},
+				AfterAll:  []ProjectBuilder{otherBuilder},
+			}
+
+			mtaPath := getTestPath("mta.yaml")
+			extPath := getTestPath("mta.mtaext")
+
+			buildParameters, messages, err := GetBuildParameters(mtaPath, []string{extPath})
+			Ω(err).Should(Succeed())
+			Ω(messages).Should(BeNil())
```

Comment:
You are checking the messages are empty afterwards, please remove one of them

Reply:
removed